benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -316,24 +316,22 @@ msgstr ""
 "To function properly, this website requires a browser that supports "
 "JavaScript. Please enable JavaScript for this website and"
 
-msgid "eligibility.pages.index.login_gov.label"
-msgstr "I am 65 years of age or older"
-
-msgid "eligibility.pages.index.login_gov.description"
-msgstr ""
-"You must be 65 years or older. This benefit does not expire, but you may "
-"need to renew. Using this benefit means your new transit fare is half of the "
-"standard fare."
-
-msgid "eligibility.pages.index.mst_cc.label"
+msgid "eligibility.pages.index.courtesy_card.label"
 msgstr "I have an MST Courtesy Card"
 
-msgid "eligibility.pages.index.mst_cc.description"
+msgid "eligibility.pages.index.courtesy_card.description"
 msgstr ""
 "This option is for people who have a current Courtesy Card or an MST RIDES "
 "Eligibility card. This benefit may need to be renewed in the future. Using "
 "this benefit means your new transit fare is half of the standard fare."
 
```

Comment:
I believe `This benefit may need to be renewed in the future. Using "
"this benefit means your new transit fare is half of the standard fare."` can be removed now. @srhhnry Is this correct? Can the expiration/renewal sentence and the half-off sentence from MST Courtesy Card be removed on this page?

<img width="622" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/93fc78f5-436a-44d8-935b-8409fc042715">